service/chat.py

```
@@ -79,6 +79,76 @@ async def completions(conversation_body:str, model_dict, api_version:str = None)
     )
     return chat_response
 
+async def generate_cmpl_id():
```

Ruff (0.8.0)

Redefinition of unused `generate_cmpl_id` from line 16

(F811)